lib/Dialect/FIRRTL/Transforms/EmitOMIR.cpp

```diff
@@ -291,7 +295,7 @@ void EmitOMIRPass::runOnOperation() {
         tracker.nla = cast<NonLocalAnchor>(tmp);
         removeTempNLAs.push_back(tracker.nla);
```

Comment:
@darthscsi This looks suspicious to me, if the tracker is already a non-local annotation, why are we deleting the NLA?  Do we end up deleting all the OMIR annotations at the end of this pass?